Resources/Views/Home/home.leaf

```diff
@@ -36,6 +36,8 @@
     <div class="container">
       #extend("Home/_about-us")
       <br/>
+      #extend("Home/_speakers")
```

Comment:
As we don't have the speakers yet, but we are about to push the tickets, this should probably go after the order tickets section.